addons/knowledge/models/knowledge_article.py

```diff
@@ -1453,15 +1453,15 @@ def _get_first_accessible_article(self):
             ], limit=1, order='sequence, internal_permission desc')
         return article
 
-    def get_valid_parent_options(self, term=""):
+    def get_valid_parent_options(self, search_term=""):
         """ Returns the list of articles that can be set as parent for the
         current article (to avoid recursions) """
         return self.search_read(
             domain=['&',
-                    ['name', '=ilike', '%%%s%%' % term],
+                    ['name', '=ilike', '%%%s%%' % search_term],
```

Comment:
Do we need to manually set the "%" around the searched term?
Also, that's a lot of "%" ^^

Reply:
Any reason to use `=ilike` and not a regular `ilike` in the first place?